Sources/LiveKit/publications/RemoteTrackPublication.swift

```diff
@@ -40,34 +89,85 @@ public class RemoteTrackPublication: TrackPublication {
         }
 
         client.sendUpdateSubscription(sid: sid,
-                                      subscribed: !unsubscribed,
-                                      videoQuality: videoQuality.toPBType())
+                                      subscribed: !unsubscribed)
     }
 
     /// disable server from sending down data for this track
     ///
     /// this is useful when the participant is off screen, you may disable streaming down their video to reduce bandwidth requirements
-    public func setEnabled(_: Bool) {
-        disabled = !disabled
-        sendTrackSettings()
+    public func setEnabled(_ enabled: Bool) {
+        self.enabled = enabled
+        //        sendTrackSettings()
```

Comment:
Remove commented out code if this is not going to be used. Or add a TODO so that somebody else reading it would know why this commented out code is here.

Reply:
😄👌